Framework/LiveData/test/KafkaEventStreamDecoderTest.h

```diff
@@ -19,20 +19,14 @@
 
 #include <Poco/Path.h>
 #include <condition_variable>
+#include <cxxtest/TestSuite.h>
 #include <iostream>
 #include <thread>
 
 using Mantid::LiveData::KafkaEventStreamDecoder;
 
 class KafkaEventStreamDecoderTest : public CxxTest::TestSuite {
 public:
-  // This pair of boilerplate methods prevent the suite being created statically
```

Comment:
In this case both tests no longer have any member vars, so the compiler does nothing. I looked at the generated .cpp for the test then used Godbolt to confirm this:

No allocation of static test suite since there is no vars associated (except for the call)
https://godbolt.org/z/FVNYU5

With member vars the compiler is forced to allocate as init (.L17 in asm)
https://godbolt.org/z/ctz5mk

Since I removed all the member vars I went ahead and removed the boilerplate as it's unnecessary. The hope being that it makes future devs pause before adding member vars (which I'm not against, provided they make the tests easier to read)

Reply:
I agree it now does nothing but I think I'd prefer if we were consistent here and had the boilerplate everywhere as it doesn't really do any harm? I don't think most devs would think twice about adding member variables as the test you're looking at makes no obvious statement about how it is constructed and the side-effects are not local.